base/docs/Docs.jl

```diff
@@ -12,7 +12,7 @@ const modules = Module[]
 
 const META′ = :__META__
 
-meta(mod) = mod.(META′)
+@eval meta(mod) = mod.$META′
```

Comment:
I think the real issue is probably some corner case in llvm 3.3 and tweaking here and there just happened to make it appears and disappears...